helmfile/values/falco.yaml.gotmpl

```diff
@@ -77,6 +79,137 @@ customRules:
+        and not plesk_writing_keys
+        and not plesk_install_writing_apache_conf
+        and not plesk_running_mktemp
+        and not networkmanager_writing_resolv_conf
+        and not run_by_chef
+        and not add_shell_writing_shells_tmp
+        and not duply_writing_exclude_files
+        and not xmlcatalog_writing_files
+        and not parent_supervise_running_multilog
+        and not supervise_writing_status
+        and not pki_realm_writing_realms
+        and not htpasswd_writing_passwd
+        and not lvprogs_writing_conf
+        and not ovsdb_writing_openvswitch
+        and not datadog_writing_conf
+        and not curl_writing_pki_db
+        and not haproxy_writing_conf
+        and not java_writing_conf
+        and not dpkg_scripting
+        and not rabbitmq_writing_conf
+        and not rook_writing_conf
+        and not php_handlers_writing_conf
+        and not sed_writing_temp_file
+        and not cron_start_writing_pam_env
+        and not httpd_writing_conf_logs
+        and not mysql_writing_conf
+        and not openvpn_writing_conf
+        and not consul_template_writing_conf
+        and not countly_writing_nginx_conf
+        and not ms_oms_writing_conf
+        and not ms_scx_writing_conf
+        and not azure_scripts_writing_conf
+        and not azure_networkwatcher_writing_conf
+        and not couchdb_writing_conf
+        and not update_texmf_writing_conf
+        and not slapadd_writing_conf
+        and not symantec_writing_conf
+        and not liveupdate_writing_conf
+        and not sosreport_writing_files
+        and not selinux_writing_conf
+        and not veritas_writing_config
+        and not nginx_writing_conf
+        and not nginx_writing_certs
+        and not chef_client_writing_conf
+        and not centrify_writing_krb
+        and not cockpit_writing_conf
+        and not ipsec_writing_conf
+        and not httpd_writing_ssl_conf
+        and not userhelper_writing_etc_security
+        and not pkgmgmt_progs_writing_pki
+        and not update_ca_trust_writing_pki
+        and not brandbot_writing_os_release
+        and not redis_writing_conf
+        and not openldap_writing_conf
+        and not ucpagent_writing_conf
+        and not iscsi_writing_conf
+        and not istio_writing_conf
+        and not ufw_writing_conf
+        and not calico_writing_conf
+        and not calico_writing_envvars
+        and not prometheus_conf_writing_conf
+        and not openshift_writing_conf
+        and not keepalived_writing_conf
+        and not rancher_writing_conf
+        and not checkpoint_writing_state
+        and not jboss_in_container_writing_passwd
+        and not etcd_manager_updating_dns
+        and not user_known_write_below_etc_activities
+        and not automount_using_mtab
+        and not mcafee_writing_cma_d
+        and not avinetworks_supervisor_writing_ssh
+        and not multipath_writing_conf
+        and not calico-node
```

Comment:
I agree with @Xartos that it would be suspicious if some openshift process started in our cluster.
But I think we should stick to the upstream as far as possible for now (like in this PR) and filter out things we don't need in a later stage.

Reply:
But from the upstream PR someone commented that you should have `calico_node` instead of `calico-node`.

Reply:
Fixed here too `calico_node` instead of `calico-node` ! thanks